website/content/terminal/usage/basics/index.md

```diff
@@ -1,114 +1,280 @@
+| survey       | Function |     No     |                                    Take a short user survey to help us improve. |
+| update       | Function |     No     |   Attempt to update (**Only for Github cloned repository installations**) |
+| wiki         | Function |    Yes    |                                   Query the Wikipedia API for a term or phrase. |
+
+### Additional Global Commands
+
+The commands listed below are not displayed on a Terminal menu, but are active globally.
+
+| Function Key |                                       Description |
+| :----------- | ------------------------------------------------: |
+| cls          |                       Clears the Terminal screen. |
+| exit         |                               Quits the Terminal. |
+| help, h, ?   |                   Prints the current menu screen. |
+| quit, q, ..  |             Navigates back one menu towards Home. |
+| reset, r     | Resets the Terminal, opening to the current menu. |
+
+## Data
+
+Many functions will require obtaining (free or subscription) API keys from various data providers.  OpenBB provides methods for consuming these data feeds but has no control over the quality or quantity of data provided to an end-user.   None are required to get started using the Terminal.  See the list of data providers [here](https://docs.openbb.co/terminal/usage/guides/api-keys), along with instructions for entering the credentials into the OpenBB Terminal.  [Request a feature](https://openbb.co/request-a-feature) to let us know what we are missing!
+
+### Default Data Sources
+
+The default data source for each function (where multiple sources are available) can be defined within the [`/sources` menu](https://docs.openbb.co/terminal/usage/guides/changing-sources).  The available sources for each function are displayed on the right of the menu, and they can be distinguished by the square brackets and distinct font color group.  Unless a preference for a particular function is defined, the command will prioritize in the order they are displayed, from left-to-right, on the Terminal screen.  To override a preference or default source, select one of the other choices by attaching the, `--source`, argument to the command syntax.  The available sources for the feature will be populated by auto complete when the `space bar` is pressed after typing `--source`.  This information is also printed with the `--help` dialogue of a command.
+
+### Importing and Exporting Data
+
+Most functions provide a method for exporting the raw data as a CSV, JSON, or XLSX file.  Exported data and user-supplied files to import are saved to the [OpenBBUserData folder](https://docs.openbb.co/terminal/usage/guides/data).  The folder is located at the root of the operating system's User Account folder.  Follow the link for a detailed description.
+
+## Charts
+
+The OpenBB charting library has themes for light and dark mode, the default state is dark.
```

Comment:
I wouldnt bother with themes here.  Maybe some intro like in other places "OpenBB returns charts yadada, here is an example of what is returned with "stoicks/load ...."